lib/ui/text.dart

```diff
@@ -423,12 +439,14 @@ class ParagraphStyle {
     TextAlign textAlign,
     FontWeight fontWeight,
     FontStyle fontStyle,
+    TextLineBreakMode textLineBreakMode,
```

Comment:
We generally try to use names inspired by CSS...  Maybe call this "textOverflow"